### DISCUSSION TITLE
Storybook makes an error "No intl context found"

### DISCUSSION THREAD

**Discussion by ktlyee** [amannn/next-intl#836] [Q&A]
### Description

I got this error when I tried to run my component that use `next-intl` in storybook.
First, I got this error
<img width="1440" alt="Screenshot 2567-02-05 at 7 08 10 PM" src="https://github.com/amannn/next-intl/assets/44608458/71b79967-49bd-40d1-aba2-833fc231da87">

After that I found this solution to set a decorator in `preview.tsx` file in `.storybook` folder.
https://next-intl-docs.vercel.app/docs/faq#can-next-intl-be-used-with-jest-and-storybook
Then I got a new error like this image
![Screenshot 2567-02-05 at 5 55 56 PM](https://github.com/amannn/next-intl/assets/44608458/69f913b9-bae4-45a1-a389-20cfcabc186d)

**Version**
 storybook: "^7.6.12"
 next-intl: "^3.5.4"

### Mandatory reproduction URL (CodeSandbox or GitHub repository)

-

### Reproduction description

Steps to reproduce:
1. run storybook in terminal
2. open 6006 port

### Expected behaviour

- `next-intl` should support storybook. 
- storybook must not make an error

**Comment by amannn** [comment]
I'll move this to a discussion since this is a usage question.

See also the previous discussions here:
- https://github.com/amannn/next-intl/discussions/771
- https://github.com/amannn/next-intl/issues?q=is%3Aissue+storybook+is%3Aclosed
